src/DelegateRegistry.sol

```diff
@@ -36,6 +36,8 @@ contract DelegateRegistry is IDelegateRegistry {
         bool success;
         unchecked {
             for (uint256 i = 0; i < data.length; ++i) {
+                // Disabling as this loop does not lead to a DOS since the registry is delegateCalling itself
+                //slither-disable-next-line calls-loop
```

Comment:
Should we disable all incorrect slither warnings with comments? Feels kinda weird to just have one. But if there are too many then the bloat isn't worth it